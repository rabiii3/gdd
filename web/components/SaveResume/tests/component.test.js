import React from 'react';
import { shallow } from 'enzyme';
import { IntlProvider } from 'react-intl';
import { SaveResumeForm } from '..';

describe('web | components | SaveResume | component', () => {
  const intlProvider = new IntlProvider({ locale: 'en', messages: {} }, {});
  const { intl } = intlProvider.getChildContext();
  describe('render', () => {
    it('should match snapshot', () => {
      const props = {
        resume: {},
        actions: {},
        classes: {},
        isEditable: false,
        setEditable: jest.fn(() => {}),
        handleIsSubmited: jest.fn(() => {}),
        isFormSubmitted: false,
        IsStatusDisabled: true,
        skills: [],
        allSkills: {},
        user:{},
      };
      const wrapper = shallow(<SaveResumeForm {...props} intl={intl} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
