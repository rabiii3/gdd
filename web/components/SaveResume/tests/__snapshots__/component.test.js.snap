// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`web | components | SaveResume | component render should match snapshot 1`] = `
<Fragment>
  <ReactFinalForm
    initialValues={
      Object {
        "status": "pending",
      }
    }
    onSubmit={[Function]}
    render={[Function]}
  />
</Fragment>
`;
